test/stream.go

```diff
@@ -0,0 +1,148 @@
+package test
```

Comment:
Is this a helper package for writing tests with `interceptor` ? Would `internal/test` make more sense maybe?